serverless/packages/functions/src/trainspace/create_trainspace.ts

```diff
@@ -0,0 +1,74 @@
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+import parseJwt from "@dlp-sst-app/core/parseJwt";
+import TrainspaceData from './trainspace-data';
+import { v4 as uuidv4 } from 'uuid';
+import { DynamoDBClient, PutItemCommand } from '@aws-sdk/client-dynamodb'; //@aws-sdk/client-dynamodb
+import { DynamoDBDocumentClient } from '@aws-sdk/lib-dynamodb';
+import { DATA_SOURCE_ARR } from '../../../../../frontend/src/features/Train/constants/trainConstants';

```

Comment:
Are you sure that this array can only come from this directory? @farisdurrani can you take a look?

Reply:
Ooffff, this is bad. You shouldn't be importing stuff from the /frontend or /backend directories. We need to consider them separate microservices because they'll be Dockerized separately. I'd recommend making a copy of the data source array in /trainspace somewhere

Reply:
@alantao912

Reply:
Okay, sure.